packages/abi-gen-templates/contract.py.handlebars

```diff
@@ -0,0 +1,44 @@
+"""Generated wrapper for {{contractName}} Solidity contract."""
+
+from typing import Optional, Tuple, Union
+
+from hexbytes import HexBytes
+from web3.datastructures import AttributeDict
+from web3.providers.base import BaseProvider
+
+from zero_ex.contract_artifacts import abi_by_name
```

Comment:
We want to make sure the templates could be used by any project. Xianny's PR (https://github.com/0xProject/0x-monorepo/pull/1883) makes it such that the templates receive the ABI as a variable. Let's use that pattern and also store the ABI as a property/static method of the wrapper.